Example/Podfile.lock

```diff
@@ -1,27 +1,27 @@
 PODS:
-  - Bolts (1.1.5):
-    - Bolts/AppLinks (= 1.1.5)
-    - Bolts/Tasks (= 1.1.5)
-  - Bolts/AppLinks (1.1.5):
+  - Bolts (1.9.0):
+    - Bolts/AppLinks (= 1.9.0)
+    - Bolts/Tasks (= 1.9.0)
+  - Bolts/AppLinks (1.9.0):
     - Bolts/Tasks
-  - Bolts/Tasks (1.1.5)
-  - Gini-iOS-SDK (0.6.0):
-    - Gini-iOS-SDK/Core (= 0.6.0)
-  - Gini-iOS-SDK/Core (0.6.0):
-    - Bolts (~> 1.1.5)
-  - Gini-iOS-SDK/Pinning (0.6.0):
-    - Bolts (~> 1.1.5)
-    - TrustKit (~> 1.5.2)
+  - Bolts/Tasks (1.9.0)
+  - Gini-iOS-SDK (1.0.0-alpha.1):
```

Comment:
Merging this into `multipage` is fine, but just remember to **not** have an alpha version when you go to `develop`.